tutorials/supported_layers.md

```diff
@@ -0,0 +1,176 @@
+| Argmax                    | √             | ×            | ×           |
+| Slicing                   | √             | ×            | ×           |
+| Eltwise                   | √             | ×            | ×           |
+| Parameter                 | √             | ×            | ×           |
+| Reduction                 | √             | ×            | ×           |
+| Silence                   | √             | ×            | ×           |
+
+### Loss Layers
+| Layer                     | Caffe         | Keras        | Tensorflow  |
+| :-----------------------: | :-----------: | :----------: | :---------: |
+| Multinomial Logistic Loss | √             | ×            | ×           |
+| Infogain Loss             | √             | ×            | ×           |
+| Softmax with Loss         | √             | ×            | √           |
+| Sum-of-Squares/Euclidean  | √             | ×            | ×           |
+| Hinge / Margin            | √             | √            | √           |
+| Sigmoid Cross-Entropy Loss| √             | ×            | √           |
+| Accuracy / Top-k layer    | √             | ×            | ×           |
+| Contrastive Loss          | √             | ×            | ×           |
+
+### Normalization Layers
+| Layer                     | Caffe         | Keras        | Tensorflow  |
+| :-----------------------: | :-----------: | :----------: | :---------: |
+| BatchNormalization        | √             | √            | √           |
+| LRN                       | √             | ×            | ×           |
+| MVN                       | √             | ×            | ×           |
+
+### Noise Layers
+| Layer                     | Caffe         | Keras        | Tensorflow  |
+| :-----------------------: | :-----------: | :----------: | :---------: |
+| GaussianNoise             | ×             | √            | √           |
+| GaussianDropout           | √             | √            | √           |
+| AlphaDropout              | √             | √            | √           |
+
+### Layer Wrappers
+| Layer                     | Caffe         | Keras        | Tensorflow  |
+| :-----------------------: | :-----------: | :----------: | :---------: |
+| TimeDistributed           | ×             | √            | √           |
+| Bidirectional             | ×             | √            | √           |
+
+### Custom Layers
+| Caffe         | Keras        | Tensorflow  |
+| :-----------: | :----------: | :---------: |
+| √             | √            | Use Keras API for custom layers |
+
+## Additional Notes:
+* Keras does not support the LRN layer used in Alexnet & many other models. To use the LRN layer refer to here: https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/keras_custom_layer_usage.md.
+* Documentation for writing your own Keras layers is found here: https://keras.io/layers/writing-your-own-keras-layers/ 
+* Caffe 3D layers can be used in Caffe 2.0
```

Comment:
We haven't tested any features for caffe 2.0 yet, have you tested this if not we can drop this line.